src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -61,15 +62,19 @@ export const TitleStatisticComponent: FunctionComponent<TitleStatisticComponent>
   onPressPrevDay,
   onPressNextDay,
 }) => {
+  const { i18nt } = useTranslate();
   return (
     <View style={styles.appHeaderWrapper}>
-      <AppText style={styles.smallText}>You distributed</AppText>
+      <AppText style={styles.smallText}>
+        {i18nt("redemptionStats", "youDistributed")}
+      </AppText>
       <AppText style={styles.statText}>{totalCount?.toLocaleString()}</AppText>
       <AppText style={styles.smallText}>
-        Last distributed at{" "}
+        {i18nt("redemptionStats", "lastDistributedAt")}{" "}
         {lastTransactionTime !== null
           ? format(lastTransactionTime, "h:mma")
-          : "-"}
+          : "-"}{" "}
```

Comment:
I think we can incorporate the spaces into the `i18nt` copies, would be easier to manage the strings that way, and would result in less maintenance at the component level. Also, this would allow us flexibility in determining which languages will display a space, and which will not.